app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ProjectsController < ApplicationController
+  def show
+    @project = Project.find(params[:project_id])
+  end
+
+  def update
+    project = Project.find(params[:project_id])
+    new_contestant = Contestant.find(params[:contestant_id])
+    ContestantProject.create(contestant: new_contestant, project: project)
```

Comment:
You can also use the params from the form to put this together. 

`ContestantProject.create(contestant_id: params[:contestant_id], project_id: params[:project_id])` for example